oton/nextflow_script.py

```diff
@@ -46,7 +46,8 @@ def build_nextflow_processes(self, ocrd_commands, dockerized=False):
         nf_processes = []
 
         for ocrd_command in ocrd_commands:
-            nextflow_process = Nextflow_Process(ocrd_command, dockerized)
+            index_pos = ocrd_commands.index(ocrd_command)
```

Comment:
This adds extra unnecessary computational complexity. Just a plain index counter should still do the job.

```
index = 0
for ocrd_command in ocrd_commands:
     ...
     index += 1
```